main/zoe/api/zoe-contract-facet.md

```diff
@@ -161,7 +162,9 @@ await zcf.saveIssuer(secondaryIssuer, keyword);
 
 Make a credible Zoe `invitation` for a smart contract. Note that `invitations` are a special case
 of an ERTP `payment`. They are in a one-to-one relationship with the `invitationIssuer`, which is used
-to validate invitations and their `amounts`.
+to validate invitations and their `amounts`. They are also in a one-to-one relationship with
+a `mint` devoted to minting new `invitations`. `zcf.makeInvitation()` serves as an interface to
```

Comment:
I wouldn't do this as two separate sentences.
"They are in a one-to-one relationship with the `invitationIssuer` and its mint, which are used
to mint invitations and validate their `amounts`."

Reply:
> Wait, it says "they"? Is it invitations or the issuer?

The only 'they's are down here, so I'm going to respond here.

I agree that one-to-one doesn't apply here. All invitations share a single unique issuer, but there are many invitations associated with many contracts. The single InvitationIssuer is tightly coupled with a single invitationMint, but only Zoe ever sees the invitationMint.

So let me change my suggestion to 

"All invitations come from a single invitation issuer and its mint, which are used
to mint invitations and validate their amounts."